scripts/cluster-install/openstack_connection.py

```diff
@@ -0,0 +1,127 @@
+import time
+import traceback
+import openstack
+
+
+def _get_client_config(config):
+    client_config = {
+        'auth_url': config['auth_url'],
+        'username': config['username'],
+        'password': config['password'],
+        'region_name': config['region_name'],
+        'project_name': config['tenant_name'],
+        'user_domain_name': 'Default',
+        'project_domain_id': 'Default'
+    }
+    return client_config
+
+
+def _get_resource_config(config, name):
+    resource_config = {
+        'name': name,
+        'image_id': '70de1e0f-2951-4eae-9a8f-05afd97cd036',  # centos image_id
```

Comment:
This should probably be in the config (it could default to this in the config file), especially as Centos+RHEL 8 are both now out so we'll hopefully upgrade at some point in the next year . <fingers crossed>

Reply:
Good point, I changed it accordingly.